### PR TITLE
Fix issues #1272 and #1287

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by DanielHeath** [comment]
https://github.com/JedWatson/react-select/issues/1272

https://github.com/JedWatson/react-select/issues/1287

**Comment by JedWatson** [comment]
This looks good, thanks @DanielHeath - I'm happy to merge it.

@jossmac can you make sure this change doesn't mess with anything you're aware of? I'm not sure why we'd avoid `float: left` but you may know something I don't.

File: less/control.less [hunk 1]
```diff
@@ -255,6 +255,7 @@
   margin: -1px;
   clip: rect(0,0,0,0);
   overflow: hidden;
+  float: left;
```
Comment (jossmac) [inline less/control.less]: @JedWatson should be fine. `position: absolute` would also work

**Comment by JedWatson** [comment]
Awesome. Thanks!

**Pull request merged**